capstone/src/main/webapp/authenticate.js

```diff
@@ -2,8 +2,9 @@ var CLIENT_ID = config.CLIENT_ID;
 var API_KEY = config.API_KEY;
 var DISCOVERY_DOCS = ["https://script.googleapis.com/$discovery/rest?version=v1"];
 var SCOPES = 'https://www.googleapis.com/auth/forms';
+var ID_TOKEN;
 
-var [dynamicButton, authorizeButton, signoutButton] = createDynamicSigninButton();
+var [dynamicButton, signoutButton] = createDynamicSigninButton();
```

Comment:
Thinking about changing this to just a signoutButton now since we don't need a sign in button on internal resources?

Reply:
Meaning that instead of dynamic button, it'll just be signout button.

Reply:
Seems reasonable if dynamic button is not used anywhere. I suppose you would have other sign in buttons (for example on the home page) just redirect to the login page?